docs/user-guide/data.md

```diff
@@ -710,6 +710,43 @@ concerning this command's use can be found in the [GCT 6.2 GridFTP User's Guide]
     data to the [JASMIN facility](https://www.jasmin.ac.uk) can be found
     in [the JASMIN user documentation](https://help.jasmin.ac.uk/article/4997-transfers-from-archer2).
 
+### Data transfer using `rclone`
+
+[Rclone](https://rclone.org/) is a command-line program to manage files on cloud
+storage. You can transfer files directly to/from cloud storage services, such as
+MS OneDrive and Dropbox. The program preserves timestamps and verifies checksums
+at all times.
+
+First of all, you must download and unzip `rclone` on ARCHER2:
+```bash
+wget https://downloads.rclone.org/v1.62.2/rclone-v1.62.2-linux-amd64.zip
+unzip rclone-v1.62.2-linux-amd64.zip
+cd rclone-v1.62.2-linux-amd64/
+```
+
+The previous code snippet uses rclone v1.62.2, which was the latest version when
+these instructions were written.
+
+Configure rclone using `./rclone config`. Tested cloud storages include:
```

Comment:
Hi @kevinstratford, I've added a sentence explaining that a `remote` setup is required. I explained that further instructions can be found on the links provided. I didn't want to duplicate the information located in the rclone guide. Please let me know if further changes are needed.

Reply:
Additional comment it would be good to add a Note in the text that if a user is seeing an issue with session timeouts that adding the -vv flag to an rclone transfer to force rclone to output to terminal as it moved data stops the timeouts killing the data transfer.

Reply:
@kevinstratford Please could you review/approve the latest changes? Thanks.